internal/search/lucky/feeling_lucky_search_job.go

```diff
@@ -233,6 +233,9 @@ func (n *notifier) New(count int) error {
 	return &alertobserver.ErrLuckyQueries{
 		ProposedQueries: []*search.ProposedQuery{{
 			Description: fmt.Sprintf("%s (%s)", n.description, resultCountString),
```

Comment:
This line should be updated to 
```suggestion
			Description: n.description
```

in tandem with https://github.com/sourcegraph/sourcegraph/pull/40877